staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiservice_controller.go

```diff
@@ -71,15 +71,18 @@ func NewAPIServiceRegistrationController(apiServiceInformer informers.APIService
 }
 
 func (c *APIServiceRegistrationController) sync(key string) error {
+	glog.Infof("APIServiceRegistrationController sync %s", key)
```

Comment:
glog with Verbose.

Reply:
This was just for debugging to understand what is actually going on in the tests. Closing this now.